src/components/Restaurant/Restaurant.jsx

```diff
@@ -1,12 +1,20 @@
+import React, { useState } from "react";
+
 const Restaurant = () => {
-  const pizzas = 0;
+  const [orders, setOrders] = useState(0);
+  const handleAddOrder = () => {
+    setOrders(orders + 1);
```

Comment:
I think nothing would change if you omit the const pizzas = 0;  Since the pizzas variable is not used anywhere in the component, removing it would simplify the code without affecting its functionality.